src/id/siimanga/build.gradle

```diff
@@ -2,8 +2,8 @@ ext {
     extName = 'Siikomik'
     extClass = '.Siikomik'
     themePkg = 'mangathemesia'
-    baseUrl = 'https://siikomik.lat'
-    overrideVersionCode = 9
+    baseUrl = 'https://siikomik.cc'
+    overrideVersionCode = 10
```

Comment:
Since there is an unresolved issue with #6775, maybe cherry-pick this change into another PR.

Reply:
> Since there is an unresolved issue with #6775, maybe cherry-pick this change into another PR.

I removed the URL change and fixed the loading of the current URL content